modules/installation-osp-kuryr-increase-quota.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+
+[id="installation-osp-kuryr-increase-quota_{context}"]
+= Increasing quota
+
+When using Kuryr SDN, you must increase quotas to satisfy the OpenStack
+resources used by Pods, Services, namespaces, and network policies.
+
+.Procedure
+
+* Increase the quotas for a project by running this command:
```

Comment:
I prefer "the following command:", but if you change I'd make sure to stay consistent within the Kuryr chapter.